controls/togglebutton/configuration.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Configuration
+page_title: .NET MAUI ToggleButton Documentation - Content Alignment
+description: Learn how to align the content of the Telerik ToggleButton for .NET MAUI horizontally or vertically.
+position: 3
+tags: .net maui, telerik toggle button for .net maui, ui for .net maui, toggle button, microsoft .net maui
+slug: togglebutton-contenfiguration
+---
+
+# .NET MAUI ToggleButton Configuration
+
+The purpose of this help article is to show you the main configuration options of the control.
+
+## Setting Content
+
+Define a content inside the ToggleButton by setting the `Content` property (`object`) or `ContentTemplate` (`DataTemplate`) property. 
+
+Here are the scenarios in which the `Content` or `ConetentTemplate` will be visualized inside the `RadToggleButton`:
+
+* If `Content` is set and `ContentTemplate` is set, the `View` returned from the `ContentTemplate.CreateView()` will be displayed inside the `RadToggleButton.ControlTemplate`, having `Content` as its `BindingContext`. 
```

Comment:
* Switch to present simple over future simple throughout the list.
* Again, consider switching to a table (as suggested in the TemplatedButton).